satpy/readers/grib.py

```diff
@@ -101,9 +102,9 @@ def _create_dataset_ids(self, keys):
             self._msg_datasets[msg_id] = ds_info
 
     @staticmethod
-    def _convert_datetime(msg, date_key, time_key, format="%Y%m%d%H%M"):
+    def _convert_datetime(msg, date_key, time_key, date_format="%Y%m%d%H%M"):
```

Comment:
Does a new linter prohibit shadowing builtins?

Reply:
flake8-bugbear I think it was. And yes, it is shadowing builtins.